tests/test_main.py

```diff
@@ -301,15 +264,12 @@ def test_initialize_api_limit(self,
         """
         # Set up return values
         mock_error = MagicMock(side_effect=Exception('API rate limit exceeded'))
-        mock_u.pagure_issues.return_value = ['mock_issue_pagure']
         mock_u.github_issues.side_effect = mock_error
 
         # Call the function
         m.initialize_issues(self.mock_config, testing=True)
 
         # Assert everything was called correctly
-        mock_u.pagure_issues.assert_called_with('key_pagure', self.mock_config)
-        mock_d.sync_with_jira.assert_any_call('mock_issue_pagure', self.mock_config)
         mock_u.github_issues.assert_called_with('key_github', self.mock_config)
```

Comment:
Do you want to change (old) line 312 to
```python
        mock_d.sync_with_jira.assert_any_call('mock_issue_github', self.mock_config)
```
or does the exception prevent the call (in which case, should we be asserting that it does _not_ get called?)?

Ditto for `test_initialize_github_error()`.